backends/xnnpack/third-party/xnnpack.buck.bzl

```diff
@@ -1055,9 +1155,13 @@ def define_xnnpack():
             "-DXNN_ENABLE_ASSEMBLY",
             "-DXNN_ENABLE_GEMM_M_SPECIALIZATION",
             "-DXNN_ENABLE_ARM_DOTPROD",
+            "-DXNN_ENABLE_CPUINFO",
+            "-DXNN_ENABLE_DWCONV_MULTIPLASS=1",
```

Comment:
do we use this?

Reply:
https://github.com/google/XNNPACK/blob/7967e2c8a6967bbe8dd4c627d07062372fadcf60/src/configs/gemm-config.c#L94

kinda dumb, but this gemm-config requires cpuinfo, but for some reason disabling cpuinfo means we will not include it into the file.

Reply:
However, we technically can turn off the dwconv_multipass